test/turnt.toml

```diff
@@ -45,3 +45,15 @@ command = "odgi flip -i {filename} --out=temp.og; odgi view -g -i temp.og"
 [envs.flip_test]
 binary = true
 command = "python3 ../slow_odgi/flip.py < {filename}"
+
+[envs.overlap_setup]
+binary = true
+command = "fn={filename}; pathsname=$(echo $fn | cut -f 1 -d '.').paths; python3 ../slow_odgi/list_paths.py < {filename} > $pathsname"
```

Comment:
Yuck! 
This is because {filename} has graph.gfa, and I'd like to generate graph.paths. 
Why? Because later, I will have a turnt command (environment "oracle") where {filename} has graph.og, and I want that turnt command to _also_ be able to look for graph.paths and find the graph.paths generated by this setup env.

Reply:
Aha! The "setup environment" trick is pretty neat; it is cool that this works.

To avoid the yuckiness about an ignored `.out` file, you can configure Turnt to put the result of this command in a `.paths` file. Just add this:

```toml
output.paths = "-"
```

Then remove the `> $pathsname` redirection, so the stuff goes to stdout. The `"-"` in the above configuration means "get your results from stdout." Turnt will then put that data in `whatever.paths`.

Reply:
Ooh many thanks; I've got this in now. Sadly the following two envs are still ugly, because I need to start with {filename} has looks like graph.gfa (resp., like graph.og) and from each of those envs, reference graph.paths.